### DISCUSSION TITLE
Overall support for Cloudflare Workers/Pages local development

### DISCUSSION THREAD

**Discussion by aydrian** [Code-Forge-Net/Remix-Dev-Tools#67] [Q&A]
Trying to use the dev tools with my app that will be deployed to Cloudflare Pages. Currently receiving the following in the browser console:
`index.js:1311 WebSocket connection to 'ws://localhost:8080/' failed`

Also, I haven't implemented the server-side set up because I'm not sure how to do it without an example. Here is the [repo for the website](https://github.com/aydrian/itsaydrian.com).

**Comment by AlemTuzlak** [comment]
Cloudflare support is now built in thanks to Remix moving to Vite so you should have no issues there
